.travis.yml

```diff
@@ -0,0 +1,45 @@
+language: python
+python:
+    2.7
+compiler: gcc # caffe
+sudo: required # for caffe
+env:
+    global:
+        - NUM_THREADS=4 # make
+        - WITH_CUDA=false # caffe
+        - WITH_CMAKE=true # caffe
+        - CAFFE_TAG="v0.11.0"
+        - CAFFE_URL="https://github.com/NVIDIA/caffe.git"
+before_install:
+    - export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib
+    - export PATH=/home/travis/miniconda/bin:$PATH # caffe
+install:
+      # caffe
+    - pushd .
+    - git clone --depth 1 --branch $CAFFE_TAG $CAFFE_URL
+    - cd caffe
+    - sudo -E ./scripts/travis/travis_install.sh
+    - cp Makefile.config.example Makefile.config
+    - sed -i 's/# CPU_ONLY/CPU_ONLY/g' Makefile.config
+    - sed -i 's/USE_CUDNN/#USE_CUDNN/g' Makefile.config
+    - make --jobs=$NUM_THREADS --keep-going all
+    - make --jobs=$NUM_THREADS --keep-going pycaffe
+    - popd
+      # change permissions for installed python packages
+    - sudo chown travis:travis -R /home/travis/miniconda
+    - for req in $(cat caffe/python/requirements.txt); do pip install $req; done
+      # DIGITS
+      # use miniconda for some deps
+    - pushd .
+    - cd caffe
+    - conda install --yes gevent greenlet
+    - popd
+    - sudo apt-get install graphviz
+    - for req in $(cat requirements.txt); do pip install $req; done
+    - for req in $(cat requirements_test.txt); do pip install $req; done
```

Comment:
Ah! It speeds up build times somewhat with pre-built packages (by about a minute, but anything helps). See [here](http://sburns.org/2014/03/28/faster-travis-builds.html).

I probably should have split that into a new pull request however.